async/models.py

```diff
@@ -21,12 +21,12 @@
 # No name 'sha1' in module 'hashlib'
 # pylint: disable=E0611
 from hashlib import sha1
-from simplejson import dumps, loads
 from traceback import format_exc
 
 from async.logger import _logger
 from async.utils import object_at_end_of_path, non_unicode_kwarg_keys
 
+import simplejson
```

Comment:
This should be up with the other library imports, not below this application's ones.